.github/workflows/ci.yaml

```diff
@@ -331,48 +331,7 @@ jobs:
           breaking: false
           push: false
           archive: true
-          # On push github.ref_name, on delete github.event.ref_name.
-          archive_labels: ${{ github.ref_name }}
-          comment: false
-  test-archive-not-exists:
```

Comment:
Remove this as labels can no longer be given to the action.

Reply:
Depending on which merges first, I suppose it would affect https://github.com/bufbuild/buf-action/pull/40, but that's okay :)